app/services/helpers/reduction-helper.js

```diff
@@ -0,0 +1,32 @@
+
+var dateFormatter = require('../date-formatter')
+
+module.exports.determineReductionsStatus = function (reductions) {
```

Comment:
Minor: Maybe rename to getReductionsByStatus as that is what is returned?

Reply:
Done